features/faq.mdx

```diff
@@ -101,4 +101,8 @@ Yes, we have achieved SOC 2 Type 2 certification. Please [contact us](mailto:hel
 
 To ensure your evaluation report reflects the newest version of your prompt template, you must configure your evaluation pipeline to use the "latest" version of the prompt template in its column step. The template is fetched at runtime, and specifying a frozen version will result in the evaluation report not reflecting your newest prompt template.
 
-![Latest FAQ Updates](/images/latest-faq.png)
\ No newline at end of file
+![Latest FAQ Updates](/images/latest-faq.png)
+
+## What model providers do you support on your evaluations page?
+
+While you can log LLM requests from any model and our Prompt Registry is agnostic, our evaluations & playground requests support OpenAI, Anthropic, Bedrock, Cohere, and Google.
```

Comment:
OpenAI, Anthropic, Bedrock, Cohere, and Google.

"OpenAI's GPT, Anthropic's Claude, Google's Gemini, Bedrock, Cohere."

Reply:
just because I think people might command-f for gemini

Reply:
Good point. Updated.